package.json

```diff
@@ -94,7 +94,7 @@
     "react-native-autocomplete-input": "^5.1.0",
     "react-native-calendars": "1.1276.0",
     "react-native-collapsible": "^1.6.0",
-    "react-native-elements": "~1.2.0",
+    "react-native-elements": "^3.4.3",
```

Comment:
another difference i found on the big tiles:

|before|after|
|---|---|
|![IMG_ABB68C178E5D-1](https://user-images.githubusercontent.com/1942953/222394575-f6014dd6-bc45-4759-8277-fb758acf53a0.jpeg)|![Simulator Screen Shot - iPhone 13 Pro Max - 2023-03-02 at 10 53 50](https://user-images.githubusercontent.com/1942953/222394592-879ce340-3cf6-41c6-a139-2311cb3d1a3d.png)|

they overlap in width a bit.

Reply:
this is a really interesting problem. actually, `ServiceTiles` has nothing to do with this update. i've searched for the problem, but I can't solve it.

Reply:
Maybe screens or wrapper or paddings changed somehow? We could inspect the screen elements to see if there are differences.

Reply:
okay, I found the cause of the problem. it is caused by prop change.
in the latest version of `rn-elements`, the `childrenContainerStyle` prop was added for the `Image` component. 
the prop we used to use as style is now called `childrenContainerStyle`. but we can still use the `style` prop. 
the `style` prop we have used so far in our code will now be the `childrenContainerStyle` prop.

Reply:
well done